shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1451,6 +1461,10 @@ - (void)updateMarkedRect:(NSDictionary*)dictionary {
 }
 
 - (void)showTextInput {
+  if (!_activeView.keyboardEnabled) {
+    [self hideTextInput];
```

Comment:
looking at the `hideTextInput`'s implementation it removes the active view from the view hierarchy and disables accessibility. Will this break voiceover? @chunhtai

Reply:
yes it will, we may need to find a better way to prevent softkeyboard from showing up. whichever the method we found here, we need to make sure the key stroke feedback in voice over still works

Reply:
You might be able to use an empty `UIView` for [`inputView`](https://developer.apple.com/documentation/uikit/uiresponder/1621092-inputview?language=objc) according to the documentation.

Reply:
And there's an `inputViewController` property as well: https://developer.apple.com/documentation/uikit/uiresponder/1621117-inputviewcontroller?language=objc

Reply:
Would either of you have a chance to give it a shot, or is there anyone else that might have time to help?

Reply:
Currently dealing with something else. I'll give that a try later.

Reply:
`inputView` didn't work as advertised for me, but `inputViewController` did. I overrode `inputViewController` like so then :
```Swift
  let inputVC = UIInputViewController()
  override var inputViewController: UIInputViewController? {
    shouldShowKeyboard ? nil : inputVC
  }
```
You get the system keyboard when returning `nil` for `inputViewController`.

Reply:
Great! Mind pushing the patch? I'll close this because I'm no iOS dev... 🙂

Reply:
Thank you for trying it out!